classes/doi/Repository.php

```diff
@@ -310,6 +309,26 @@ public function depositAll(Context $context)
         }
     }
 
+    /**
+     * Checks whether a DOI object is referenced by ID on any pub objects
+     */
+    public function isAssigned(int $doiId): bool
```

Comment:
Do we need to implement this function in apps, e.g. for issues (ojs), files (omp)?

Reply:
Added!